kyo-caliban/src/main/scala/kyo/resolvers.scala

```diff
@@ -0,0 +1,65 @@
+package kyo
+
+import caliban.*
+import caliban.interop.tapir.*
+import caliban.interop.tapir.TapirAdapter.*
+import kyo.internal.KyoSttpMonad
+import sttp.tapir.Codec.JsonCodec
+import sttp.tapir.Endpoint
+import sttp.tapir.capabilities.NoStreams
+import sttp.tapir.server.ServerEndpoint
+import sttp.tapir.server.netty.*
+import zio.*
+import zio.stream.ZStream
+
+opaque type Resolvers = Aborts[Throwable] & ZIOs
```

Comment:
FYI the direct syntax has a bug that crashes the compiler if effects are defined as `opaque`. I think it's ok, though. We should fix the direct syntax instead.